modape/scripts/modis_collect.py

```diff
@@ -149,35 +180,45 @@ def cli(src_dir: str,
     collected = []
 
     for group in groups:
-        group_pattern = re.compile(group + '.*hdf')
+        group_pattern = re.compile(group + ".*hdf")
         group_files = [str(x) for x in hdf_files if group_pattern.match(x.name)]
         group_files.sort()
 
-        _raw_h5 = ModisRawH5(files=group_files,
-                            targetdir=targetdir,
-                            vam_product_code=vam_code,
-                            interleave=interleave)
+        _raw_h5 = ModisRawH5(
+            files=group_files, targetdir=targetdir, vam_product_code=vam_code, interleave=interleave
+        )
 
         if last_collected is not None:
 
             if not _raw_h5.exists:
-                raise ValueError("Output H5 %s does not exist! Can't check last-collected!" % _raw_h5.filename)
+                raise ValueError(
+                    "Output H5 %s does not exist! Can't check last-collected!" % _raw_h5.filename
+                )
 
             last_collected_infile = _raw_h5.last_collected
 
             if not last_collected_infile:
                 raise ValueError(f"No last_collected recorded in {_raw_h5.filename}")
 
             if not last_collected == last_collected_infile:
-                raise ValueError(f"Last collected date in file is {last_collected_infile} not {last_collected}")
+                raise ValueError(
+                    f"Last collected date in file is {last_collected_infile} not {last_collected}"
+                )
 
-        to_process.update({
-            group: {
-                "raw_h5": _raw_h5,
-                "compression": compression,
-                "force": force,
+        to_process.update(
+            {
+                group: {
+                    "raw_h5": _raw_h5,
+                    "compression": compression,
+                    "force": force,
+                    "create_only": (
+                        False
+                        if tiles_required is None
+                        else group.split(".")[2].strip("*") not in tiles_required
+                    ),
```

Comment:
Not-required tiles are merely created (initialized .h5 file), no data should be ingested